tests/simulators/webots/dynamic/webots_data/controllers/scenic_supervisor/scenic_supervisor.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from controller import Supervisor
+
+import scenic
+from scenic.simulators.webots import WebotsSimulator
+
+WEBOTS_RESULT_FILE_PATH = f"{os.path.dirname(__file__)}/../../../results.txt"
+
+
+def send_results(data):
+    with open(WEBOTS_RESULT_FILE_PATH, "w") as file:
+        file.write(data)
+
+
+supervisor = Supervisor()
+simulator = WebotsSimulator(supervisor)
+
+path = supervisor.getCustomData()
+print(f"Loading Scenic scenario {path}")
+scenario = scenic.scenarioFromFile(path)
+
+scene, _ = scenario.generate()
+simulation = simulator.simulate(scene, verbosity=2)
```

Comment:
Maybe add a large upper bound to the time limit so if the test fails, it does so gracefully?